index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Landing page</title>
+    <link rel="stylesheet" href="./resources/css/style.css"/>
+</head>
+<body>
+    <nav>
+        <div class="heading">
+            <img class="logo"
+            src="./resources/images/36.jpg" alt="logo image"/>
+             <h2 class="nav_heading">FavCrunchies</h2>
```

Comment:
Correct this indentation it is supposed to start at same point with the image tag since they are both siblings

Reply:
okay Sir.